src/main.rs

```diff
@@ -59,11 +60,11 @@ fn main() {
     let write_opt = cli::gen_write_options(&args);
     let cleaning_opt = cli::gen_cleaning_options(&args);
 
-    let in_file  = args.value_of("in-file").unwrap();
-    let out_file = args.value_of("out-file").unwrap();
+    let in_file = args.value_of("in-file").unwrap();
+    let out_file = args.value_of("out-file").unwrap_or("");
```

Comment:
I think we should use `Option<String>` in this case. So `.unwrap_or("")` is not needed.